docs/formats.md

```diff
@@ -52,64 +87,64 @@ Creates a CSS file with variable definitions based on the style dictionary
 
 * * *
 
-### scss/variables
+### scss/variables 
 
 
 Creates a SCSS file with variable definitions based on the style dictionary
 
-**Example**
+**Example**  
 ```scss
 $color-background-base: #f0f0f0;
 $color-background-alt: #eeeeee;
 ```
 
 * * *
 
-### scss/icons
+### scss/icons 
 
 
 Creates a SCSS file with variable definitions and helper classes for icons
 
-**Example**
+**Example**  
 ```scss
 $content-icon-email: '\E001';
 .icon.email:before { content:$content-icon-email; }
 ```
 
 * * *
 
-### less/variables
+### less/variables 
```

Comment:
not important, just a heads up: I see some diffs are just a trailing space (added) at the end of the line

Reply:
Yeah, this part of the file is generated by JSDoc. Not sure what causing the difference there.

Reply:
👍